python/interpret_community/explanation/explanation.py

```diff
@@ -518,7 +518,7 @@ def _perf_dict(self, y, y_hat, i):
         di = {}
         di["actual"] = y[i]
         di["predicted"] = y_hat[i]
-        if isinstance(y[i], str) or isinstance(y_hat, str):
+        if _is_type(y[i], y_hat[i], str) or _is_type(y[i], y_hat[i], bool) or _is_type(y[i], y_hat[i], np.bool_):
```

Comment:
Because `isinstance` takes a tuple you can simplify this to if _is_type(y[i], y_hat[i], (str, bool, np.bool_)):`

Reply:
Or you could remove the helper entirely with `if any(isinstance(v, (str, bool, np.bool_)) for v in [y[i], y_hat[i]]):`

Reply:
done!